core/src/main/kotlin/org/sert2521/sertain/Robot.kt

```diff
@@ -51,6 +53,9 @@ suspend fun robot(configure: RobotScope.() -> Unit) {
         }
     }
 
+    val mc = MotorController(TalonId(0))
```

Comment:
This probably shouldn't be here, because ID 0 is the always the power distribution board.

Reply:
Hahaha what this didn't happen